docs/releases/1.10.txt

```diff
@@ -496,3 +500,12 @@ removed in Django 1.10 (please see the :ref:`deprecation timeline
 
 * The warning that :class:`~django.core.signing.Signer` issued when given an
   invalid separator is now a ``ValueError``.
+
+* ``django.forms.widgets.Media.absolute_path`` removed
```

Comment:
We don't usually document private API removals unless we have evidence they are commonly used. I don't think that applies to these two items. (Also, the section you are in here is for features that went through the deprecation cycle.)